dev/CommonStyles/AppBarButton_themeresources.xaml

```diff
@@ -104,6 +104,8 @@
     <Thickness x:Key="AppBarButtonInnerBorderCompactMargin">2,6,2,22</Thickness>
     <Thickness x:Key="AppBarButtonInnerBorderOverflowMargin">4,0,4,0</Thickness>
 
+    <x:String x:Key="AppBarButtonFlyoutGlyph">&#xE974;</x:String>
```

Comment:
FYI this resource being here, instead of in the theme dictionaries, means you wont be able to override it from app.xaml

Reply:
OK, I have moved it.